src/constants/routes.js

```diff
@@ -363,6 +374,25 @@ export const routes = {
 export const pathNames = mapObjectToObject(routes, route => route.path);
 export const validLinks = mapObjectToObject(routes, route => route.links);
 export const hideNavbar = mapObjectToObject(routes, route => !!route.hideNavbar);
+export const showSubNavbar = mapObjectToObject(routes, route => !!route.showSubNavbar);
+
+export const getCurrentMatch = (location) => {
+    const links = Object.keys(pathNames);
+    const paths = Object.values(pathNames);
```

Comment:
You can iterate properly using Object.entries

Reply:
old code did not change